### PR TITLE
Support campaigns of type "Display".

### PR DESCRIPTION
**Pull request by ekoval**
Closes: #30

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: sitewit/services.py [hunk 1]
```diff
@@ -335,6 +362,30 @@ def cancel_campaign_subscription(self, account_token, campaign_id,
...
+        Returns:
+            Please see response format here:
+            https://sandboxpapi.sitewit.com/Help/Api/
+            DELETE-api-subscription-cancel-campaign-display
+        """
+        data = {'campaignId': campaign_id,
+                'cancelType': 'Immediate' if immediate else 'EndOfCycle'}
+
+        return self.delete(
+            'api/subscription/cancel/campaign/display/', data=data,
```
Comment (RayeN) [inline sitewit/services.py]: We're still passing the data as a form data.  We should probably switch to JSON here as well. (and above).

File: tests/campaigns/test_integration.py [hunk 2]
```diff
@@ -34,11 +39,45 @@ def assert_sub_returned(self, response, campaign_id, budget,
         self.assertEqual(response['subscription']['budget'], budget)
 
 
+class SearchMixin(object):
+    campaign_type = 'search'
+
+    @classmethod
+    def subscribe_method(cls):
```
Comment (RayeN) [inline tests/campaigns/test_integration.py]: We could probably make this as class level attr (the same as with `campaign_type`).  Just a bit more concise.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: tests/campaigns/test_integration.py [hunk 3]
```diff
@@ -34,11 +54,20 @@ def assert_sub_returned(self, response, campaign_id, budget,
         self.assertEqual(response['subscription']['budget'], budget)
 
 
+class BaseSearchCampaignSubscriptionTestCase(BaseSubscriptionTestCase):
+    campaign_type = 'search'
+
+
+class BaseDisplayCampaignSubscriptionTestCase(
+        BaseSubscriptionTestCase):
+    campaign_type = 'display'
```
Comment (RayeN) [inline tests/campaigns/test_integration.py]: We should probably define `campaign_type` where it's needed - think it will be more concise

**Comment by RayeN** [comment]
:+1: The comment is minor.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Pull request merged**